src/elements/tag/tag-group/tag-group.ts

```diff
@@ -58,7 +58,7 @@ class SbbTagGroupElement extends SbbNamedSlotListMixin<SbbTagElement, typeof Lit
    * If set multiple to true, the value is an array.
    */
   @property()
-  public set value(value: string | string[] | null) {
+  public set value(value: string | (string | null)[] | null) {
```

Comment:
Looks weird, do we expect that?

Reply:
I just kept the same type of the getter because it seems that the manifest takes the prop types from the getter (see the other comment on the tag-group about nullability)